OpenOversight/app/commands.py

```diff
@@ -313,9 +313,7 @@ def process_salary(row, officer, compare=False):
                 if Decimal('%.2f' % salary.salary) == Decimal('%.2f' % float(row['salary'])) and \
                         salary.year == int(row['salary_year']) and \
                         salary.is_fiscal_year == is_fiscal_year and \
-                        ((salary.overtime_pay and 'overtime_pay' in row and
-                            Decimal('%.2f' % salary.overtime_pay) == Decimal('%.2f' % float(row['overtime_pay']))) or
-                            (not salary.overtime_pay and ('overtime_pay' not in row or not row['overtime_pay']))):
+                        ((salary.overtime_pay and 'overtime_pay' in row and Decimal('%.2f' % salary.overtime_pay) == Decimal('%.2f' % float(row['overtime_pay']))) or (not salary.overtime_pay and ('overtime_pay' not in row or not row['overtime_pay']))):
```

Comment:
This only appears in the diff because of a formatting change.  Since I didn't write the logic, I'm not the right person to add an explanatory comment.

Reply:
This is comparing the imported salary with the officer's existing salaries, due to type and formatting differences.